pkgs/love2d/example/assets/modern_pong/states/inGame.lua

```
@@ -0,0 +1,249 @@
+    love.graphics.draw(player2.mesh)
+
+    -- Draw the game ball
+    love.graphics.setColor(0.75, 0.75, 0.75) 
+    love.graphics.circle('fill', gameBall.x, gameBall.y, gameBall.size)
+
+    love.graphics.setColor(1, 1, 1)
+
+    -- Draw the dashed center line
+    local dashHeight = 10
+    local dashSpace = 15
+    local centerY = love.graphics.getWidth() / 2 
```

Luacheck (1.2.0)

[warning] line contains trailing whitespace

(W612)